### PR TITLE
Updated Token Hook API and OIDC API with correct aud and sub reserved…

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Clarified conflicting doc on `aud` and `sub` reserved ID Token claims in API reference.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-405347](https://oktainc.atlassian.net/browse/OKTA-405347)

### CODE REVIEW

**Review by quyle-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/token-hook/index.md [hunk 1]
```diff
@@ -473,38 +480,41 @@ This `add` operation adds `lax` to the end of the array. Alternatively, you can
    ]
 }
 ```
-> **Note:** Attempting to add an element within an array that doesn't exist or specifying an invalid index results in the entire PATCH failing and errors logged in the token hooks events. 
+
+> **Note:** Attempting to add an element within an array that doesn't exist or specifying an invalid index results in the entire PATCH failing and errors logged in the token hooks events.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/token-hook/index.md:484]:
```suggestion
> **Note:** If you attempt to add an element within an array that doesn't exist or specify an invalid index, the entire PATCH will fail and errors will be logged in the token hooks events.
```

File: packages/@okta/vuepress-site/docs/reference/token-hook/index.md [hunk 2]
```diff
@@ -473,38 +480,41 @@ This `add` operation adds `lax` to the end of the array. Alternatively, you can
...
 
 ### Sample Response to Replace an Existing Claim
 
 You can modify existing custom claims or OIDC standard profile claims, such as `birthdate` and `locale`. You can't, however, modify any system-specific claims, such as `iss` or `ver`, and you can't modify a claim that isn't currently part of the token in the request payload. Attempting to modify a system-specific claim or using an invalid operation results in the entire PATCH failing and errors logged in the token hooks events.
 
-For the list of access token reserved claims that you can't modify, see [Access Tokens Scopes and Claims](/docs/reference/api/oidc/#access-token-scopes-and-claims). Note that although the `aud` and `sub` claims are listed as reserved claims, you can modify those claims in access tokens. You can't modify these claims in ID tokens.
+For the list of access token reserved claims that you can't modify, see [Access Tokens Scopes and Claims](/docs/reference/api/oidc/#access-token-scopes-and-claims).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/token-hook/index.md:490]: For cross-reference text to links, the style guide says to word it like this: See...for...

File: packages/@okta/vuepress-site/docs/reference/token-hook/index.md [hunk 3]
```diff
@@ -557,66 +570,69 @@ The resulting JSON object:
...
           }
       ]
 }
 ```
+
 ### Sample Response to Remove Token Claims
-You can remove existing custom claims or OIDC standard profile claims, such as `birthdate` or `locale`. You can't, however, remove any system-specific claims, such as `iss` or `ver`, and you can't remove a claim that isn't currently part of the token in the request payload. Attempting to remove a system-specific claim or using an invalid operation results in the entire PATCH failing and errors logged in the token hooks events. 
 
-For the list of access token reserved claims that you can't remove, see [Access Tokens Scopes and Claims](/docs/reference/api/oidc/#access-token-scopes-and-claims). 
+You can remove existing custom claims or OIDC standard profile claims, such as `birthdate` or `locale`. You can't, however, remove any system-specific claims, such as `iss` or `ver`, and you can't remove a claim that isn't currently part of the token in the request payload. Attempting to remove a system-specific claim or using an invalid operation results in the entire PATCH failing and errors logged in the token hooks events.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/token-hook/index.md:606]:
```suggestion
You can remove existing custom claims or OIDC standard profile claims, such as `birthdate` or `locale`. You can't, however, remove any system-specific claims, such as `iss` or `ver`, and you can't remove a claim that isn't currently part of the token in the request payload. If you attempt to remove a system-specific claim or use an invalid operation, the entire PATCH will fail and errors will be logged in the token hooks events.
```

File: packages/@okta/vuepress-site/docs/reference/token-hook/index.md [hunk 4]
```diff
@@ -557,66 +570,69 @@ The resulting JSON object:
...
+
+For the list of access token reserved claims that you can't remove, see [Access Tokens Scopes and Claims](/docs/reference/api/oidc/#access-token-scopes-and-claims).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/token-hook/index.md:608]:
```suggestion
See [Access Tokens Scopes and Claims](/docs/reference/api/oidc/#access-token-scopes-and-claims) for the list of access token reserved claims that you can't remove.
```

File: packages/@okta/vuepress-site/docs/reference/token-hook/index.md [hunk 5]
```diff
@@ -625,40 +641,43 @@ See [ID Token Claims](/docs/reference/api/oidc/#id-token-claims) for a list of I
...
       "sjc",
       "lax",
       "sfo",
       "oak"
    ]
 }
 ```
-You can remove the element from the array by specifying the array name followed by the index of the element that you want to remove. You don't need to specify a value for the `remove` operation. But, you can specify `null` as the value if you want. 
+
+You can remove the element from the array by specifying the array name followed by the index of the element that you want to remove. You don't need to specify a value for the `remove` operation. But, you can specify `null` as the value if you want.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/token-hook/index.md:656]:
```suggestion
You can remove the element from the array by specifying the array name followed by the index of the element that you want to remove. You don't need to specify a value for the `remove` operation, but you can specify `null` as the value if you want.
```

**Review by christinechoy-okta — COMMENTED** [review]
Just a few things...

**Review by christinechoy-okta — APPROVED** [review]

**Pull request merged**